AISKU/src/AISku.ts

```diff
@@ -89,7 +89,12 @@ const defaultConfigValues: IConfigDefaults<IConfiguration|IConfig> = {
             [_eInternalMessageId.SdkLdrUpdate]:cfgDfMerge<IThrottleMgrConfig>(default_throttle_config),
             [_eInternalMessageId.CdnDeprecation]:cfgDfMerge<IThrottleMgrConfig>(default_throttle_config)
         }
-    )
+    ),
+    extensionConfig: cfgDfMerge<{[key: string]: any}>({
+        ["AppInsightsCfgSyncPlugin"]: cfgDfMerge<ICfgSyncConfig>({
+            cfgUrl: CONFIG_ENDPOINT_URL
```

Comment:
This is our settings, the endpointURL set in CfgSyncPlugin should be the "user settings"

Reply:
and version for this should be 3.2.0